### PR TITLE
added italian translations, and translated array,select and token widget

### PR DESCRIPTION
**Pull request by giuliaghisini**
Added the ability to translate the placeholder and 'No options' string for ArrayWidget, SelectWidget, TokenWidget. 
Added some italian translations

### CODE REVIEW

File: src/components/manage/Widgets/ArrayWidget.jsx [hunk 1]
```diff
@@ -32,13 +32,17 @@ const CreatableSelect = loadable(() => import('react-select/creatable'));
 
 const messages = defineMessages({
   select: {
-    id: 'Select…',
-    defaultMessage: 'Select…',
+    id: 'Select...',
```
Comment (tisto) [inline src/components/manage/Widgets/ArrayWidget.jsx:35]: I think we never used this consistently in Plone core. I was wondering if there is anybody we can ask how to use the "..." properly in translations...

Comment (sneridagh) [reply src/components/manage/Widgets/ArrayWidget.jsx:35]: I vaguely remember @albertcasado in favor of using the “unified” single character for it.

Comment (tisto) [reply src/components/manage/Widgets/ArrayWidget.jsx:35]: Then we might want to keep using the unified single character. If I am not mistaken this is used most of the time in the Plone translations anyways.

@giuliaghisini would you be ok with changing "..." to the unified character? Maybe someone steps up to go through the Volto core and replaces all occurrences of "...".

Comment (giuliaghisini) [reply src/components/manage/Widgets/ArrayWidget.jsx:35]: ok, yes!

**Comment by giuliaghisini** [comment]
@tisto @sneridagh updated translations and changed three dots with single char!

**Comment by tisto** [comment]
@giuliaghisini thank you! i18n was outdated. I pushed an update. Let's see if CI is green now.

**Comment by tisto** [comment]
@giuliaghisini pls run "yarn run i18n" again :)

**Comment by giuliaghisini** [comment]
> @giuliaghisini pls run "yarn run i18n" again :)

sotty, done now!

**Pull request merged**